scripts/src/submission/submission.py

```diff
@@ -518,7 +518,7 @@ def get_file_type(file_path):
     return "unknwown", None
 
 
-def download_index_data(repository, branch="gh_pages"):
+def download_index_data(repository: str, branch: str = "gh-pages") -> dict:
```

Comment:
I assume this fix is specific to your submission module, and does not exist in the source that this reimplements? As in, there's no bug in the original code?

Reply:
The original code in checkpr doesn't use any default value. So this only for the submission module